src/tutorial/styles.scss

```diff
@@ -31,6 +31,7 @@
       margin-top: spacing.$spacing-03;
       color: $color-blue-60-2;
       font-size: 1rem;
+      @include type.type-style('body-compact-02');
```

Comment:
I think font size also included with the `body-compact-02`. Can you please check?

Reply:
Yes that's unnecessary. will remove it.